src/docs/markdown/caddyfile/directives/abort.md

```diff
@@ -0,0 +1,32 @@
+---
+title: abort (Caddyfile directive)
+---
+
+# abort
+
+Immediately and forcefully closes the connection without writing a response. Also interrupts any other HTTP streams on the same connection.
+
+
+## Syntax
+
+```caddy-d
+abort [<matcher>]
+```
+
+## Examples
+
+Forcefully close a connection received for unknown domains when using a wildcard certificate:
```

Comment:
This is a good example, better than the one I pushed. Would be OK keeping this. Make sure it's not `caddy-d` though since it's a whole Caddyfile.